backend/schema/corpora_schema.md

```diff
@@ -198,8 +133,8 @@ publication\_doi|DOI of the associated journal publication.
 
 #### Presentation Hints
 
-The metadata fields below are fully optional. They aren't needed for integration, and Corpora can present the data fine without them, but if they are
-present Corpora will do something with them. This allows submitters to fine-tune how their datasets are presented, which is a common request.
+The metadata fields below are fully optional. They aren't needed for integration, and cellxgene can present the data fine without them, but if they are
+present cellxgene will do something with them. This allows submitters to fine-tune how their datasets are presented, which is a common request.
```

Comment:
Nits:

_~fully~ optional_ 

 _**present** the data ... are **present** cellgene will_ 

_do something with them_ ;-)

Reply:
:+1: